### ISSUE TITLE
Since v2.2.0: Cannot read property 'opener' of null when $.parseHTML and then get css style

### ISSUE THREAD

**Issue by todorpr** [jquery/jquery#2866]
I just upgraded to jQuery v2.2.0 and this problem appeared.

I could reproduce it in the following case:
- parse some html using $.parseHTML
```javascript
var el = $.parseHTML('<span style="font-family: Cuprum,sans-serif; font-size: 14px; color: #999999;">some text</span>');
```
- on the already parsed html wrap it in jQuery container:
```javascript
var span = $(el[0]);
```
- now try to get css style on the jQuery element:
```javascript
var style = span.css("font-size");
console.log(style);
```
Which results in:
```javascript
Uncaught TypeError: Cannot read property 'opener' of null
```

jsBin: http://jsbin.com/ruyecuxogu/edit?html,js,console,output

**Comment by todorpr** [comment]
I found out that the problem comes from this function
https://github.com/jquery/jquery/blob/99e8ff1baa7ae341e94bb89c3e84570c7c3ad9ea/src/css/var/getStyles.js

if change:
```javascript
if (!view.opener ) {
    view = window;
}
```
to:
```javascript
if (!view || !view.opener ) {
    view = window;
}
```
the error disappears and getting css style is done properly.

**Comment by markelog** [comment]
Hm, it seems issue is relevant for the beta too

**Comment by gibson042** [comment]
This is caused by cfe468f29c4cbe1a457d0feb17dec90dcfd7c280, right?

```js
document.implementation.createHTMLDocument( "" ).defaultView
// → null
```

**Comment by timmywil** [comment]
@gibson042 Must be, but are you implying that `createHTMLDocument` isn't worth it? Current solution is to default to the current window, which might even be alright when building append-able elements for iframes.

**Comment by gibson042** [comment]
I'm not implying that, just trying to establish the blast radius. The fix proposed in #2867 seems fine.
